code/__DEFINES/obj_flags.dm

```diff
@@ -37,6 +37,12 @@
 #define EXAMINE_SKIP (1<<15) // Makes the Examine proc not read out this item.
 #define XENOMORPH_HOLDABLE (1<<16) // A Xenomorph can hold this item.
 #define NO_PIXEL_RANDOM_DROP (1<<17) //if dropped, it wont have a randomized pixel_x/pixel_y
+/// Items that go above hair without hiding it when worn, for the few cases where it's necessary (needs to be implemented in most cases)
+/// I hate the fact I had to put it here, but since any item can be worn anywhere, it's necessary.
+#define ABOVE_HAIR_WORN (1<<18)
```

Comment:
why? alternate_worn_layer exists

Reply:
Where?

Reply:
Oh, I'm stupid, wow.

Reply:
So, I tried using `glasses?.alternate_worn_layer ? glasses.alternate_worn_layer : HEAD_LAYER` and it seems to be producing very strange effects, some items leave behind their mutable appearance, some others are spawned on the wrong layer, it's a bit of a mess.